core/src/main/java/org/elasticsearch/rest/action/cat/RestNodesAction.java

```diff
@@ -85,7 +85,7 @@ public RestChannelConsumer doCatRequest(final RestRequest request, final NodeCli
         clusterStateRequest.clear().nodes(true);
         clusterStateRequest.local(request.paramAsBoolean("local", clusterStateRequest.local()));
         clusterStateRequest.masterNodeTimeout(request.paramAsTime("master_timeout", clusterStateRequest.masterNodeTimeout()));
-
+        final boolean fullId = request.paramAsBoolean("full_id", false);
```

Comment:
because that is the most obvious way to do this. I like when thinks are obvious especially when you look where this is consumed you don't need to look for it, it's right where I'd expect it.